examples/Getting Started/Export Image.ipynb

```diff
@@ -0,0 +1,223 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import ee\n",
+    "ee.Initialize()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exporting images, map tiles, features, and videos\n",
+    "\n",
+    "Shows how to export assets using Google Earth Engine API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# prepare export parameters\n",
+    "dem = ee.Image(\"JAXA/ALOS/AW3D30_V1_1\")\n",
+    "region = ee.Geometry.LineString([[93.11668395996094, 28.709710285906034], [93.1486988067627, 28.673946616818256]]).bounds()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "RPZYAKKYWM4HCWQR7E6ILJTM\n"
+     ]
+    }
+   ],
+   "source": [
+    "# schedule a new export image task\n",
+    "file_name = 'dem'\n",
+    "export_region = region.getInfo()['coordinates'] # BUG: it should just accept region as-is\n",
+    "task = ee.batch.Export.image.toDrive(dem, description=file_name, fileNamePrefix=file_name, region=export_region, scale=30)\n",
+    "task.start()\n",
+    "\n",
+    "# remember task id and use it to check tasks status later\n",
+    "task_id = task.id\n",
+    "print(task_id)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'creation_timestamp_ms': 1516292370542,\n",
+       " 'description': 'dem',\n",
+       " 'id': 'RPZYAKKYWM4HCWQR7E6ILJTM',\n",
+       " 'state': 'READY',\n",
+       " 'task_type': 'EXPORT_IMAGE',\n",
+       " 'update_timestamp_ms': 1516292370542}"
+      ]
+     },
+     "execution_count": 31,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# create a new task using id\n",
+    "task = ee.batch.Task(task_id)\n",
+    "\n",
```

Comment:
Suggest separating out the task status query, because you may want to query a single task multiple times.